docs/collections/_schema/human-readable-schema.md

```diff
@@ -0,0 +1,226 @@
+
+```cedar
+entity User in [Group] {
+    personalGroup: Group,
+    delegate?: User,
+    blocked: Set<User>,
+};
+```
+
+Note that, unlike the JSON schema format, human-readable schema syntax allows you to declare multiple entity types that share the same definition using a single declaration. For example, `entity UserA, UserB, UserC ...` declares entity types `UserA`, `UserB`, and `UserC` that all have the same membership relations and shapes.
+
+### Membership relations {#schema-entitytypes-memberOf}
+
+The parent types are specified by `in <EntityTypes>` after `entity <EntityName>`, where `EntityTypes` can be a list of entity type names surrounded by brackets and delimited by `,`, or an entity type name if there is only one parent type. For example, you can also declare entity type `User` like `entity User in Group ...`. The membership relation declaration is optional, whose absence means that the declared entity type does not have any parent entity types. 
+
+### Shape {#schema-entitytypes-shape}
+
+You specify the shape of an entity type using the [record syntax](../policies/syntax-datatypes.html#datatype-record) of Cedar policies. That is, attribute declarations are enclosed by brackets, each of which is a `<Name>:<Type>` pair. Attribute names are either identifiers or strings. Such a declaration also defines a [record schema type](#schema-entitytypes-shape-record), which we will visit later in the document. To make entity type declarations consistent with [common type declarations](#schema-commonTypes), users can write a `=` before attribute declarations like `entity User = {...};`.
+
+Note that if you omit attribute declarations, then entities of this type do not have any attributes. This is equivalent to specifying an empty record (i.e., `{}`).
+
+
+### Schema types {#schema-types}
+
+Schema types can be used as RHS of an attribute or common type declaration. Cedar data types have their corresponding schema types: The corresponding type names of Cedar primitive data types [Boolean](../policies/syntax-datatypes.html#datatype-boolean), [String](../policies/syntax-datatypes.html#datatype-string), [Long](../policies/syntax-datatypes.html#datatype-string) are `Bool`, `String`, `Long`, respectively. An entity type or an extension type is specified by its name: The entity type name is either an identifier or identifiers separated by `::`. For example, both `User` and `ExampleCo::User` are both valid entity type names. An extension type name is just an identifier, specifically, `ipaddr` or `decimal` as of now. Since Cedar verison 3.1, we reserve the namespace `__cedar` and allow fully-qualified type names for primitive and extension types with this namespace. For example, `__cedar::ipaddr` uniquely identifies the `ipaddr` extension type.
```

Comment:
What's RHS?

Reply:
right-hand-side. common enough jargon, but worth changing for the docs